replication/replication.proto

```diff
@@ -213,3 +231,27 @@ message GetVolumeReplicationInfoResponse {
   // This field is REQUIRED.
   .google.protobuf.Timestamp last_sync_time = 1;
 }
+// Specifies what source the replication will be created from. One of the
+// type fields MUST be specified.
+message ReplicationSource {
+  // Message containing volume id
+  message VolumeSource {
+    // Contains identity information for the existing volume.
+    // This field is REQUIRED.
+    string replication_volume_id = 1;
+  }
+
+  // Message containing volume group id
+  message VolumeGroupSource {
+    // Contains identity information for the existing volume group.
+    // This field is REQUIRED.
+    string replication_volume_group_id = 1;
```

Comment:
This should be renamed as volume_group_id? No need to append replication to it.

Reply:
Renamed to volume_group_id